src/components/posts/PostList.tsx

```diff
@@ -1,12 +1,19 @@
-import "../../assets/styles/Posts/Posts.css";
-import Post from "./Post";
-import { useState, useContext } from "react";
-import { AiOutlineSearch as Search } from "react-icons/ai";
-import { PostsContent } from "../../pages/FeedPage";
+import '../../assets/styles/Posts/Posts.css'
+import Post from './Post'
+import { useState } from 'react'
+import { AiOutlineSearch as Search } from 'react-icons/ai'
+import postAtom from './State'
+import { useRecoilValue } from 'recoil'
+
+export interface IPost {
```

Comment:
Would be a bit cleaner to move this into a `types.ts` file (can be the same one as for the IComment interface)